app/src/main/java/org/p2p/wallet/newsend/model/TemporaryAccount.kt

```diff
@@ -6,21 +6,17 @@ import kotlinx.parcelize.Parcelize
 import org.p2p.solanaj.core.PublicKey
 import org.p2p.wallet.utils.toPublicKey
 
+private const val SEND_LINK_FORMAT = "t.key.app/"
```

Comment:
I would prefer to add this in Gradle strings or in xml strings and use same way to use one resource

for case if we will rename it for example